### DISCUSSION TITLE
Can't "un-stale" an issue by commenting on it

### DISCUSSION THREAD

**Discussion by oliviernguyenquoc** [microcks/microcks#1137] [Q&A] [answered]
Hi, 

I think I did not understand how to unstale an issue.
I though that commenting on it would unstale it (by the "Manage stale issues and PRs " Github Action): https://github.com/microcks/microcks/issues/951

Should I create a new issue ? Sounds weird.

**Comment by lbroudoux** [comment, marked as answer]
Hi @oliviernguyenquoc

Commenting an issue prevents from having the `stale` tag applied but doesn't automatically un-stale it once the tag is there. You need a maintainer to put the `keep-open` tag instead of re-opening the issue. That's what I've done on #951. Having a look at it.

Cheers,

**Reply by oliviernguyenquoc** [reply]
Ok. My mistake. Next time, I open a new issue and link to the old one, I guess
